scripts/new_project.kts

```diff
@@ -189,13 +190,25 @@ object NewProject {
         showMessage("=> 🔎 Renaming package name within files...")
         File(projectPath)
             .walk()
-            .filter { it.name.endsWith(".kt") || it.name.endsWith(".xml") }
+            .filter {
+                it.name.endsWith(".kt")
+                        || it.name.endsWith(".xml")
+                        || it.name.endsWith(".gradle.kts")
```

Comment:
Could you please help recheck if these lines are indented correctly 🙏🏼

Reply:
I don't like it too, but nothing after I auto-format 🙈 

What about you?

Reply:
Mine look like this after auto-format 🙈 

![Screen Shot 2565-08-16 at 10 45 27](https://user-images.githubusercontent.com/13492460/184793721-656bebba-5431-43cd-9366-6f88708a5842.png)

Reply:
Conclusion: From our Compass, we will use 8 space indent for line wraps, therefore, the current indent is correct. Thanks @Tuubz 

<img width="635" alt="Screen Shot 2565-08-16 at 10 53 41" src="https://user-images.githubusercontent.com/13492460/184794837-d299a588-bb14-48db-ad57-ac683b6feaf5.png">

Reply:
[Optional] If it really bothers you, we could write a helper method for it 🤔 
```kotlin
private fun String.endsWithAny(vararg suffixes: String): Boolean {
    for (suffix in suffixes) if endsWith(suffix) return true
    return false
}

File(projectPath)
    .walk()
    .filter { it.name.endsWithAny(".kt", ".xml", ".gradle.kts") }
```

Reply:
@ryan-conway Updated 🤖